www/common/make-backup.js

```diff
@@ -169,10 +169,18 @@ define([
                 data: fData
             });
         }
-
-        var href = (fData.href && fData.href.indexOf('#') !== -1) ? fData.href : fData.roHref;
-        var parsed = Hash.parsePadUrl(href);
-        if (['pad', 'file'].indexOf(parsed.hashData.type) === -1) { return; }
+        var href;
+        var parsed;
+        var linkRegex = new RegExp("^(http|https)://");
+        if (fData.href && linkRegex.test(fData.href)) {
```

Comment:
This *should* work but this isn't a very clean way to detect if this is a link. If someone created a pad at the very beginning of CryptPad and this pad's href wasn't properly migrated to the new format, it's possible that they still have a pad with an absolute URL in their drive `https://cryptpad.fr/pad/#...`.
It's also possible that in the future we'll want to have absolute URL for pads stored at a different place.

Another way to detect if the item is a link is to simply check if `channel` exists:
```javascript
if (!fData.channel) {
  // this is a link
} else {
```